build/linux/installer/conf/telegraf-rs.conf

```diff
@@ -73,11 +73,11 @@
 
   ## Logging configuration:
   ## Run telegraf with debug log messages.
-  debug = false
+  debug = true
```

Comment:
have set this to true until I scale test. Will change this back to non debug mode before merging this to ci_prod.